### ISSUE TITLE
Follow system theme addon for website dark mode and customizable colors

### ISSUE THREAD

**Issue by Chiroyce1** [ScratchAddons/ScratchAddons#4041]
### Is your request related to a problem? Please describe.

Not really, it would be nice if the theme could automatically change depending on your Operating System's theme, like GitHub does.

### Describe the solution you'd like

In the "website dark mode and customizable colors" addon, add a checkbox to automatically switch to preset themes according to the system theme.

### Describe alternatives you've considered

You could make it a separate addon, I'm just suggesting it, so feel free to add a few more points.

_this is also my first ever issue on GitHub_

**Comment by mxmou** [comment]
> In the "website dark mode and customizable colors" addon, add a checkbox to automatically switch to preset themes according to the system theme.

This would be more useful if custom presets were implemented first. A UI for choosing the preferred light and dark presets would also be necessary.

**Comment by Chiroyce1** [comment]
> This would be more useful if custom presets were implemented first. A UI for choosing the preferred light and dark presets would also be necessary.

Yeah that would be better - but for now, at least the Scratch Addons presets can be added, and then later, custom presets for dark and light themes.

**Comment by GrahamSH-LLK** [comment]
> > This would be more useful if custom presets were implemented first. A UI for choosing the preferred light and dark presets would also be necessary.
> 
> Yeah that would be better - but for now, at least the Scratch Addons presets can be added, and then later, custom presets for dark and light themes.
> 
> Also I actually didn't add the "status: needs triage" label, it was automatically added since I chose the feature request template.

Yeah, that's the point. mxmou triaged it and added labels, so now needs triage is untrue

**Comment by Chiroyce1** [comment]
> Yeah, that's the point. mxmou triaged it and added labels, so now needs triage is untrue

Oh lol - I didn't know that just reviewing it was a triage, thanks for letting me know!

**Comment by Chiroyce1** [comment]
Now that this has not been implemented in over 4 months - I decided to give a go at trying to implement this, and I instantly have a question, should it depend on a background SA thread that checks it via a media query then accordingly modifies the JSON settings to automatically set it to dark/light - or add an "automatic" value to the settings, and evaluate it to dark or light when the pages are loaded?
